### DISCUSSION TITLE
[Bug] 个人主页动态显示为“鉴权失败，请联系账号组”

### DISCUSSION THREAD

**Discussion by SuanOrange** [ywmoyue/biliuwp-lite#524] [bug]
### Bug 描述

扫码登陆后动态页无内容，显示“鉴权失败，请联系账户组”，有时显示正常内容

### 操作步骤

```plain text
1.打开应用(版本为v4.6.27.0)
2.点击右上角头像，显示“个人中心”
3.点击“个人中心”，显示“动态”按钮
4.点击“动态”，显示“鉴权失败，请联系账号组”
```

### 期望结果

正常显示个人空间动态

### 截图

![Screenshot 2024-03-13 182033](https://github.com/ywmoyue/biliuwp-lite/assets/67503405/de446d6a-98ac-49e8-99b4-93ffba8ef3ee)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 22H2 (Build 19045)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

个人账户动态显示，其他账户动态也会显示
有概率会显示正常内容，但是大概率显示为“鉴权失败，请联系账号组”
频繁刷新会导致应用卡死崩溃

### 代码贡献

我更擅长发现问题

**Comment by forchannot** [comment]
最新版4.6.29仍然出现了这个问题，重新安装，二维码登录，验证码登录都尝试过了也不行 @ywmoyue
```
2024-04-11 16:51:08.6427|INFO|2|StackTrace.LogRequest|网络请求: [GET]https://api.vc.bilibili.com/dynamic_svr/v1/dynamic_svr/dynamic_new?type_list=268435455&uid=170023310&access_key={hasValue}&sign={hasValue}|
2024-04-11 16:51:08.6877|ERROR|2|StackTrace.GetDynamicItems|鉴权失败，请联系账号组|
```

**Reply by ywmoyue** [reply]
这是另一个问题：首页动态Tab页打开失败
两个页面用的是不同的接口
